themes/theme-b2c-tailwind/src/atoms/index.css

```diff
@@ -1,12 +1,13 @@
-@import "./badge.css";
```

Comment:
[Off-topic] Everybody who touches this file changes the quotes. I must've seen it a few times in previous PRs.
😂

Reply:
Isn't it a problem with our rules?

Reply:
@danzanzini we don't have any project standard regard single and double quotes, so in general, single quotes are preferred